opacus/grad_sample/grad_sample_module.py

```diff
@@ -98,8 +98,8 @@ def __getattr__(self, item):
                 return submodules[item]
             raise e
 
-    def forward(self, x, *args, **kwargs):
-        return self._module(x, *args, **kwargs)
+    def forward(self, *args, **kwargs):
+        return self._module(*args, **kwargs)
```

Comment:
`x` in completely unnecessary, and fails for BERT models from huggingface, which only accept named parameters.

`nn.Module` signature doesn't have `x`